.github/workflows/docker.build.yml

```diff
@@ -56,4 +54,42 @@ jobs:
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.docker_meta.outputs.labels }}
+      #Staging: Create pull request on deploy repo from a merged PR
+      -
+        name: create pull request for deploy on staging
+        uses: actions/checkout@v3
+        with:
+          repository: wmde/wbaas-deploy
+      # Make changes to pull request here
+      - run: |
+          get_tag=$(echo ${{ steps.docker_meta.outputs.tags }} | cut -d':' -f2)
+          yq -i '.image.tag=$get_tag' k8s/helmfile/env/local/ui.values.yaml.gotmpl
+          yq -i '.image.tag=$get_tag' k8s/helmfile/env/staging/ui.values.yaml.gotmpl
+      - uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
+          title: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
+          branch: deploy-to-staging-${{ steps.docker_meta.outputs.tags }}
```

Comment:
Seems this breaks because the branch name would now contain colons, which is not allowed.

We probably can use `deploy-to-staging-${get_tag}` instead which should be unique as well (we'd need to make it available across steps or use something similar though).

Sorry for not catching this in my initial review.